src/routes/+page.svelte

```diff
@@ -1,20 +1 @@
-<script lang="ts">
-	import * as Card from '$lib/components/ui/card';
-	import { Button } from '$lib/components/ui/button';
-</script>
-
-<div class="col-span-2 row-span-2">
-	<Card.Root class="size-full text-white flex flex-col">
-		<Card.Header>
-			<!--TODO: Replace this with i18n key-->
-			<Card.Title>Changelog:</Card.Title>
-		</Card.Header>
-		<Card.Content class="flex-1">
-			<p class="whitespace-pre-line">Changelog TXT</p>
-		</Card.Content>
-		<Card.Footer class="flex-1 items-end justify-end">
-			<Button variant="default" class="rounded-xl border-2 border-gray-600">More</Button>
-			<Button variant="link" class="rounded-xl border-2 border-gray-600 ml-5">Hide</Button>
-		</Card.Footer>
-	</Card.Root>
-</div>
+src/routes/homepage/+page.svelte
```

Comment:
Homepage only accessable at `/homepage` route instead of `/` route.
`/` serves this string `src/routes/homepage/+page.svelte`

Reply:
> Homepage only accessable at `/homepage` route instead of `/` route. `/` serves this string `src/routes/homepage/+page.svelte`
@maalni 
I was planning on doing the cards as is in one file for now and then throwing the individual cards in the homepage folder and moving the other logic in the root page (@MasterLegend2772 's idea), as Im still working on the cards, it's easier to keep them together for now

Reply:
Then add link to /homepage in root layout for now